src/types/AuditLogEvent.ts

```diff
@@ -12,4 +12,143 @@ type AuditLogEvent = {
   user?: string
 }
 
-export default AuditLogEvent
+export enum AuditLogEventSource {
+  CoreHandler = "CoreHandler",
+  EnrichWithPncQuery = "EnrichWithPncQuery"
+}
+
+export type AuditLogEventOption = {
+  code: string
+  type: string
+}
+
+export const AuditLogEventOptions = {
+  HEARING_OUTCOME_RECEIVED_PHASE_1: {
```

Comment:
I think the json keys should be camelCase here to be consistent and follow the naming convention.

Reply:
Yeah that was a lazy copy-paste from old repo, hoped I can get away with it 😄

Reply:
Fixed in [430b13e](https://github.com/ministryofjustice/bichard7-next-core/pull/423/commits/430b13e0464f5641b7cc638c61cd8f1f5e636e7f)